.github/workflows/run_forklift_build_on_kind.yml

```diff
@@ -33,10 +33,14 @@ jobs:
           path: ~/.cache/bazel
           key: ${{ runner.os }}-bazel-cache
 
+      - run: sudo apt-get install -y nbdkit-plugin-dev
+
       - name: Get kubectl
         run: curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
       - run: chmod u+x kubectl
 
+      - run: cd forklift/ && bazel sync --configure
```

Comment:
hmmm not sure if the `sync` should be here or in the forklift repo.
But if it will stay here, it should be specified also in the action.yml.

Reply:
i'll remove this line from this GH-Action , since we are not building it per PR.